apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/index.php

```diff
@@ -388,9 +388,9 @@ function wpcom_premium_global_styles_is_site_exempt( $blog_id = 0 ) {
 
 	switch_to_blog( $blog_id );
 
-	$note = 'See https://wp.me/p7DVsv-fY6#comment-44778';
+	$note = 'Automated sticker. See https://wp.me/p7DVsv-fY6#comment-44778';
 
-	add_blog_sticker( 'wpcom-premium-global-styles-exemption-checked', $note, null, $blog_id );
+	add_blog_sticker( 'wpcom-premium-global-styles-exemption-checked', $note, 'a8c', $blog_id );
```

Comment:
To clarify: any non-empty string will avoid logging a user. Just went with `'a8c'` because it felt _cleaner_ than other options like `' '`, or `'something else'`

Reply:
[non-blocking] Since we are invoking this in 4 different places, what do you think if we create a function that adds this sticker with these specific parameters? Another approach could be to refactor `a8c` to a constant so that if we ever want to change the user we don't need to change it x times.

Reply:
Two of the current 4 usages will be removed after the A/B test ends. But I liked the `a8c` constant approach so I can add a clarifying comment: https://github.com/Automattic/wp-calypso/pull/80048/commits/62862fa9e7e990910002c22cfbfa0fc0528a4405